core/state_transition.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+var (
+	Big0 = big.NewInt(0)
```

Comment:
There's already a `common.Big0`. Shouldn't we try and consolidate these scattered big constants somewhere?